core/server/services/stats/lib/mrr-stats-service.js

```diff
@@ -0,0 +1,154 @@
+const {DateTime} = require('luxon');
+
+class MrrStatsService {
+    constructor({db}) {
+        this.db = db;
+    }
+
+    /**
+     * Get the current total MRR, grouped by currency (ascending order)
+     * @returns {Promise<MrrByCurrency[]>}
+     */
+    async getCurrentMrr() {
+        const knex = this.db.knex;
+        const rows = await knex('members_stripe_customers_subscriptions')
+            .select(knex.raw(`plan_currency as currency`))
+            .select(knex.raw(`SUM(
+                CASE WHEN plan_interval = 'year' THEN
+                    plan_amount / 12
```

Comment:
You are right. But maybe we should round the total instead of the individual calculations.

If you have 1,000 members that pay a subscription of $100 yearly:
* **Rounding inside the query**: You would get $8.33 * 1,000 = $8,330 MRR. But in fact 12 * 8,330 would give an ARR of $99,960, while it should be $100,000. 
* **Rounding at the end**: ROUND(1000*100/12), being $8,333.33 might be more correct. Giving a ARR of $99,999.96

Reply:
Whichever makes most sense - the values are stored in cents so we'll only ever be at most $0.01/mo per user away from the correct value (0.05% in this example) so I don't think it matters 😄